lib/mayaHydra/mayaPlugin/renderOverride.cpp

```diff
@@ -1096,6 +1096,22 @@ void MtohRenderOverride::_InitHydraResources(const MHWRender::MDrawContext& draw
     _dirtyLeadObjectSceneIndex = MAYAHYDRA_NS::MhDirtyLeadObjectSceneIndex::New(_inputSceneIndexOfFilteringSceneIndicesChain);
     _inputSceneIndexOfFilteringSceneIndicesChain = _dirtyLeadObjectSceneIndex;
 
+#ifdef MAYA_HAS_VIEW_SELECTED_OBJECT_API
```

Comment:
Moved from _CreateSceneIndicesChainAfterMergingSceneIndex().  The isolate select scene index is independent of display style and therefore does not need to be re-created on shading change (e.g. bounding box mode).